code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -83,6 +104,38 @@
 /obj/item/weapon/gun/energy/laser/cyborg/emp_act()
 	return
 
+////////Multistate Weapons///////////
+
+/obj/item/weapon/gun/energy/laser/hybrid
+	multistate = 1
+	name = "hybrid laser gun"
+	icon_state = "laser_hybrid"
+	item_state = ""
+	desc = "An advanced laser weapon that is able to increase and decrease its beam for either non-lethal and lethal combat."
+	origin_tech = "combat=4;materials=4;powerstorage=4"
+	ammo_type = list(/obj/item/ammo_casing/energy/laser, /obj/item/ammo_casing/energy/disabler)
+
+/obj/item/weapon/gun/energy/laser/hybrid/attack_self(mob/living/user as mob) //For multistate processing
```

Comment:
as mob is pointless since you're already typecasting the argument "user" as a mob/living